src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -31,10 +31,15 @@ import { sidebarStyles } from '../Sidebar.styles';
 import { isJsonString } from './sample-query-utils';
 
 export class SampleQueries extends Component<ISampleQueriesProps, any> {
+  resetCollapse: boolean;
+  groups: any[];
   constructor(props: ISampleQueriesProps) {
     super(props);
+    this.resetCollapse = true;
+    this.groups = [];
```

Comment:
Looks like `groups` is a class property and also a state property. Is there a use for the first one?

Reply:
On taking a second look, the two variables `groups` and `resetCollapse` are not necessary, since the `selectedQuery` state variable is present

Reply:
8c67a81bfe47a5011eb99bb9edb1028dd5591c7f